sentry-android-core/src/main/java/io/sentry/android/core/AppLifecycleIntegration.java

```diff
@@ -69,9 +69,8 @@ public void register(final @NotNull IScopes scopes, final @NotNull SentryOptions
         options
             .getLogger()
             .log(
-                SentryLevel.INFO,
-                "androidx.lifecycle is not available, AppLifecycleIntegration won't be installed",
-                e);
+                SentryLevel.WARNING,
```

Comment:
Unrelated to this PR, but I just noticed while testing no androidx.lifecycle case that it's noisy and decided to change that